src/upgrade/index.js

```
@@ -0,0 +1,71 @@
+ *   3. plpgsql-resolve   — DROP+CREATE plpgsql per DB to refresh .so path
+ *   4. env-refresh       — regenerate ~/.autopg/<app>.env URLs
+ *   5. consumer-signal   — touch ~/.autopg/state/upgrade.signal
+ *   6. health-validate   — pg_isready + per-DB plpgsql smoke test
+ *
+ * Patches the upgrade-path hole left by autopg-v22 partial roll-out:
+ * users running pgserve@2.1.x → autopg@2.2.x get transparent migration.
+ *
+ * See: .genie/wishes/autopg-upgrade-command/WISH.md
+ */
+
+const { runStep } = require('./runner');
```

ESLint

[error] 'require' is not defined.

(no-undef)